molgenis-data-elasticsearch/pom.xml

```diff
@@ -55,5 +55,10 @@
             <artifactId>protonpack</artifactId>
             <version>1.8</version>
         </dependency>
+        <dependency>
+            <groupId>org.molgenis</groupId>
+            <artifactId>molgenis-mail</artifactId>
```

Comment:
elasticsearch no longer needs to depend on molgenis-mail, so I've removed these lines
molgenis-mail still is a rather small module, it depends on molgenis-data so is a bit higher than what I'd expect in molgenis-util or molgenis-core
Could move the mail classes to molgenis-data but that's already a bit beefy class.
Or start a molgenis-settings package. Unsure.